crates/tx5-connection/src/hub.rs

```diff
@@ -225,4 +258,9 @@ impl Hub {
             Err(Error::other("already connected"))
         }
     }
+
+    /// Set a new webrtc config.
+    pub fn set_webrtc_config(&self, webrtc_config: Vec<u8>) {
+        *self.webrtc_config.lock().unwrap() = webrtc_config;
```

Comment:
Can this be changed once there are already connections?
Does it just apply to new connections and not existing ones?

Might be useful to have a couple of these details in the doc comment?

Reply:
done: https://github.com/holochain/tx5/pull/92/commits/f724b0f018ce04e619bb2b2c381f9191bd8f5158